### PR TITLE
[FLOC 4269] Rename keypair suggestion

### PR DESCRIPTION
**Pull request by davidoliverSP2**
Fixes 4269

Addresses a suggestion raised by @moshez 

Note - Point 2 raised in 4269 "Move the Volume Hub part of step 2 to Step 1.5. Looks too mushed up with Step 2." has not been corrected in this PR - this will be done as part of the review comments submitted by FFFunction, that are currently being transcribed into JIRA tickets. (will update here when references are available)

### CODE REVIEW

File: docs/docker-integration/cloudformation.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ Once it's up and running we'll guide you through a tutorial to deploy a sample a
 		</div>
 		<div class="step-stages__step">
 			<img src="../_images/02-create-key.png" alt="Creating a new AWS key pair"/>
-            <span>Click "Create Key Pair". Give your key pair a meaningful name, like <strong>flocker-test</strong>. You'll need this later, so <strong>make a note of it</strong>.</span>
+            <span>Click "Create Key Pair". Give your key pair a meaningful name, like <strong>username-flocker-test</strong>. You'll need this later, so <strong>make a note of it</strong>.</span>
```
Comment (moshez) [inline docs/docker-integration/cloudformation.rst]: Maybe we should put "username" in italics, to indicate that this is a "meta variable" and not the literal word "username"?

File: docs/docker-integration/cloudformation.rst [hunk 2]
```diff
@@ -165,7 +165,7 @@ Once it's up and running we'll guide you through a tutorial to deploy a sample a
                 <img src="../_images/31-stack-outputs.png" alt="Stack outputs in CloudFormation" style="margin: 2em 0;"/>
                 <p>Now open a Terminal window, and run the following commands to log in and verify your cluster is working.</p>
                 <p>Where a command includes a string like <code>&lt;ClientNodeIP&gt;</code>, use the corresponding value from the Outputs tab.</p>
-                <p>Where a command has <code>&lt;KeyPath&gt;</code> use the path on your machine to the <code>.pem</code> file you downloaded in Step 1, for example: <code>~/Downloads/flocker-test.pem</code>.</p>
+                <p>Where a command has <code>&lt;KeyPath&gt;</code> use the path on your machine to the <code>.pem</code> file you downloaded in Step 1, for example: <code>~/Downloads/username-flocker-test.pem</code>.</p>
```
Comment (moshez) [inline docs/docker-integration/cloudformation.rst]: Ditto

**Comment by davidoliverSP2** [comment]
Hi @moshez 

Thanks for the feedback - we use <example> to display variables throughout the rest of the docs, so I've added these, and made them italic as suggested - I thought that helped make them easily identifiable in this doc as user specific variables.

**Comment by wallrj** [comment]
Thanks @davidoliverSP2 

I quickly skimmed http://clusterhq-staging-docs.s3-website-us-east-1.amazonaws.com/cloudformation-docimprovements-FLOC-4269/docker-integration/cloudformation.html and it looks better.

I might have suggested using `<em>` rather than `<i>` but that's just nit picking.

Please merge.

**Pull request merged**